src/sidebar/components/Annotation/AnnotationActionBar.tsx

```diff
@@ -118,7 +118,7 @@ function AnnotationActionBar({
   };
 
   return (
-    <div className="flex text-xl" data-testid="annotation-action-bar">
+    <div className="flex text-[16px]" data-testid="annotation-action-bar">
```

Comment:
This container has several `IconButton` children and this will cause all of their icons (which get sized to `1em`) to be the desired 16px square. But as in the previous example with Spinner, I don't want these icon button sizes to change if the type size token values change in Tailwind.